pkgs/applications/science/biology/febio-studio/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, stdenv, fetchFromGitHub, cmake, gcc, qt515, zlib, libGL, libGLU, openssl, libssh, xorg, wrapQtAppsHook, qtbase}:
```

Comment:
```suggestion
{ lib, stdenv, fetchFromGitHub, cmake, gcc, qt515, zlib, libGL, libGLU, openssl, libssh, xorg, wrapQtAppsHook, qtbase }:
```

Reply:
is this specific version of qt needed? `qt515` also seems to be the default, so why not just `qt5`?

Reply:
It is the recommended version from the developers